sdk/avalon_sdk/work_order/work_order_params.py

```diff
@@ -125,8 +130,10 @@ def set_session_key_iv(self, session_iv):
 
     def set_requester_nonce(self, requester_nonce):
         """Set requesterNonce work order parameter."""
-        if not requester_nonce.strip():
-            raise TypeError("requesterNonce cannot be None")
+        if requester_nonce is None or \
+                not is_valid_hex_str(requester_nonce):
+            raise ValueError("Empty or Invalid data \
+format for requesterNonce")
```

Comment:
Use this form instead (similar to C++) to continue strings:

```
raise ValueError("Empty or Invalid data "
    "format for requesterNonce")
```

Reply:
done